src/tests/math/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+ADDTEST(math)
```

Comment:
The `ADDTEST` macro doesn't allow `fypp` preprocessing at the current state, due to
https://github.com/fortran-lang/stdlib/blob/c48da8fb67abb9238fcfad82b7855c9b0b2e2c6c/src/tests/CMakeLists.txt#L1-L7

Reply:
I used `.f90` version of the same `fypp` file to resolve this issue.